ducktape/template.py

```diff
@@ -31,7 +31,8 @@ def render_template(self, template, **kwargs):
         """
         if not hasattr(template, 'render'):
             template = Template(template)
-        ctx = dict(self.__class__.__dict__)
+        ctx = {k: getattr(self, k) for k in dir(self.__class__)}
+        ctx.update(dict(self.__class__.__dict__))
```

Comment:
Is anything extra being captured here vs what is covered by `dir` collecting props up the class hierarchy? If yes, this case doesn't seem to handle multiple levels of inheritance; if not, we can just remove this.